test/conftest.py

```diff
@@ -136,8 +137,8 @@ def _upload_dicom_instance(dicom_dir: Path, **kwargs: Any) -> None:
 @pytest.fixture(scope="session")
 def _setup_pixl_cli(ftps_server: PixlFTPServer, _populate_vna: None) -> Generator:
     """Run pixl populate/start. Cleanup intermediate export dir on exit."""
-    # CLI calls need to have CWD = test dir so they can find the pixl_config.yml file
     run_subprocess(["pixl", "populate", str(RESOURCES_OMOP_DIR.absolute())], TEST_DIR)
+    run_subprocess(["pixl", "populate", str(RESOURCES_OMOP_DICOMWEB_DIR.absolute())], TEST_DIR)
```

Comment:
If this works then that's all good, may need to have a wait between the two populate commands so they don't clash. Expect that you may get a condition where the PIXL project name private tag is changed before it becomes stable so the ftps export wouldn't happen perhaps?